docs/SEND_MESSAGE_USING_ETHEREUM_PUBLIC_KEY.md

```diff
@@ -0,0 +1,73 @@
+# Send message only knowing recipient's Ethereum Public Key
+
+## Variables
+
+- `A` is Alice's Ethereum root HD public key
+- `B` is Bob's Ethereum root HD public key
+- `a` is the private key of `A`, and is only known by Alice,
+- `b` is the private key of `B`, and is only known by Bob.
+
+## Properties
+
+a. Alice knows Bob's Ethereum root HD public key `B`
+b. Alice wants to send message `M` to Bob using waku,
+c. Bob can find `M` from waku store,
+d. Bob can recognize he is `M`'s recipient when relaying it via waku relay,
+e. Carole cannot read `M`'s content even if she is storing it or relaying it.
+
+### Out of scope
+
+The solution **may not** not have the following properties:
+
+i. Carole must not know Bob is the recipient of a message
+ii. Bob must not be able to deduce Alice's Ethereum root HD public key `A`
+
+## Solution
+
+1. Alice derives Bob's waku public Key `Bw` from `B`,
+2. Alice derives her own waku public key `Aw` from `A`,
+3. Alice creates `M'` containing `M` and `Aw`,
+4. Alice encrypts `M'` using `Bw`, resulting in `m'`,
+5. Alice creates waku message `Mw` with `payload` `m'` and `contentTopic` `/recipient_key/1.0.0/Bw`, 
+   with `Bw` in hex format (`0xAb1..`)
+6. Alice publish `Mw` on default pubsub topic,
+7. If Bob receives `Mw` via waku relay, he recognises `Bw` in the `contentTopic` field,
+8. Otherwise, when Bob goes online he queries waku store nodes settings `contentTopics` to `["/recipient_key/1.0.0/Bw"]`, to retrieve `Mw`,
```

Comment:
Small detail but I'd probably conform to existing content topic schema with encoding and namespacing etc

Reply:
Sorry, I don't understand what schema, encoding and namespacing you are referring to. Can you share a link please ?

Reply:
See below, couldn't find in spec directly, inspired by eth2 spec https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/p2p-interface.md

Reply:
I think it'd be great if we could define some guidelines for app developers to follow in our specs (See #114).

Reply:
Good point, do you want to create an issue in rfc repo? We could have a section like https://rfc.vac.dev/spec/10/#recommendations-for-clients - not quite sure which spec this would belong in... maybe its own doc? not sure